crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -893,6 +896,76 @@ impl TypedTransaction {
     }
 }
 
+impl From<RpcTransaction> for TypedTransaction {
```

Comment:
can we make this TryFrom and return alloy::ConversionError

Reply:
updated